macOS/DuckDuckGo/Bookmarks/View/BookmarksEmptyStateView.swift

```diff
@@ -28,9 +28,9 @@ private struct ButtonWidthPreferenceKey: PreferenceKey {
 
 private struct DynamicWidthButtonStyle: ButtonStyle {
     let maxWidth: CGFloat?
-    let defaultBackgroundColor: DesignSystemColor
-    let pressedBackgroundColor: DesignSystemColor
-    let textColor: DesignSystemColor
+    let defaultBackgroundColor: SharedDesignSystemColor
+    let pressedBackgroundColor: SharedDesignSystemColor
+    let textColor: SharedDesignSystemColor
```

Comment:
Since `DesignSystemColor` is now unavailable on he mac, this change was required

Reply:
You can make a typealias for macOS, to prevent changing this now and again after the palettes are unified cross-platform:
`typealias DesignSystemColor = SharedDesignSystemColor`